metagraph/tests/annotation/test_converters.cpp

```diff
@@ -210,8 +210,8 @@ TEST(RowDiff, succ) {
 
         sdsl::int_vector_buffer succ(succ_file, std::ios::in);
         ASSERT_EQ(expected_succ.size(), succ.size());
-        for (uint32_t i = 0; i < succ.size(); ++i) {
-            EXPECT_EQ(expected_succ[i], succ[i]) << max_depth << " " << i;
+        for (uint32_t i = 0; i < expected_succ.size(); ++i) {
+            EXPECT_EQ(expected_succ[i] + 1, graph->rank_node(succ[i] + 1)) << max_depth << " " << i;
```

Comment:
how about we replace the `+1`s with `AnnotatedDBG::anno_to_graph_index` so it's more clear what's going on here?